### DISCUSSION TITLE
Retry logic

### DISCUSSION THREAD

**Discussion by sergebagi** [allure-framework/allure2#2451] [Questions & Support] [answered]
hi team,
I'm building the content of the allure-results folder from scratch as my test framework is not supported.
I noticed that if a test is run several times it shows up in Retries which is fine but it seems the test run that is put in the Overview tab is not always the 1st run. For instance I run the same test 3 times: Run#1 goes to Retries tab (I expect it to be in Overview tab), Run#2 goes to Overview tab (I expect this one to go to Retries tab) and Run#3 goes to Retries (this is fine).
How do I control this such a way that Run#1 is always placed in the Overview tab ?
I did not find anything in the doc.
Interestingly, I tried with the Playwright plugin and I see the same issue. Is that a bug ?

**Comment by delatrie** [comment, marked as answer]
Hi, @sergebagi !

Test results are ordered by start time. The latest one goes into the overview, the rest are shown on the "Retries" tab. I've just checked allure-playwright, and it behaves exactly like that.

Could you please share the code of the test, how exactly you rerun it, and what the test result files look like?

**Reply by sergebagi** [reply]
You're right, it's always the latest run that goes to Overview (in my initial question I thought it's random but it's not). I would have thought the 1st run goes to overview and subsequent runs go to Retries. 
Looks like it's by design, so I'm good. Thanks for the quick reply.
